### PR TITLE
[14.0][MIG] Account Brand

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**

### CODE REVIEW

**Review by sbejaoui — APPROVED** [review]

**Comment by sbejaoui** [comment]
please rebase on 14.0

**Comment by AmmarOfficewalaSerpentCS** [comment]
@sbejaoui Please Review

File: account_brand/tests/test_account_analytic_move.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            {
+                "name": "Test sale",
+                "code": "XX_700",
+                "user_type_id": self.env.ref("account.data_account_type_revenue").id,
+            }
+        )
+        self.move = self.env["account.move"].create(
+            {
+                "partner_id": self.env.ref("base.res_partner_12").id,
+                "type": "out_invoice",
```
Comment (sbejaoui) [inline account_brand/tests/test_account_analytic_move.py]:
```suggestion
                "move_type": "out_invoice",
```

File: account_brand/tests/test_account_move.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        self.partner_id = self.env.ref("base.res_partner_12")
+        self.partner_id.property_account_receivable_id = self.account_receivable
+        type_revenue = self.env.ref("account.data_account_type_revenue")
+        self.account_revenue = self.env["account.account"].create(
+            {"name": "Test sale", "code": "XX_700", "user_type_id": type_revenue.id}
+        )
+        self.move = self.env["account.move"].create(
+            {
+                "partner_id": self.partner_id.id,
+                "type": "out_invoice",
```
Comment (sbejaoui) [inline account_brand/tests/test_account_move.py]:
```suggestion
                "move_type": "out_invoice",
```

File: account_brand/tests/test_account_move.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        account = self._get_receivable_account(self.move)
+        self.assertEqual(
+            account,
+            self.account_receivable_partner_brand_default,
+        )
+        move = self.env["account.move"].create(
+            {
+                "partner_id": self.partner_id.id,
+                "brand_id": self.brand_id.id,
+                "type": "out_invoice",
```
Comment (sbejaoui) [inline account_brand/tests/test_account_move.py]:
```suggestion
                "move_type": "out_invoice",
```

File: account_brand/tests/test_brand_mixin.py [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+            view_type="form",
+        )
+        doc = etree.XML(view["arch"])
+        self.assertTrue(doc.xpath("//field[@name='brand_use_level']"))
+
+    def test_reverse_move(self):
+        move = self.env["account.move"].create(
+            {
+                "name": "Sample invoice",
+                "type": "out_invoice",
```
Comment (sbejaoui) [inline account_brand/tests/test_brand_mixin.py]:
```suggestion
                "move_type": "out_invoice",
```

**Review by sbejaoui — CHANGES_REQUESTED** [review]
Please check Travis

**Comment by patrickrwilson** [comment]
@AmmarOfficewalaSerpentCS It looks like the error showing in the tests happens in runbot too. Just trying to open the invoicing app causes the error to display:
image

**Comment by Freni-OSI** [comment]
> @AmmarOfficewalaSerpentCS It looks like the error showing in the tests happens in runbot too. Just trying to open the invoicing app causes the error to display:
> image

Fixed here #82 .

**Comment by patrickrwilson** [comment]
@AmmarOfficewalaSerpentCS @Freni-OSI can you squash the automated commits that are showing in the history? (Translations, .pot, readme upd, etc) 
The error is fixed so this should be good for reviewers after you clean those.

**Comment by patrickrwilson** [comment]
@AmmarOfficewalaSerpentCS @Freni-OSI It seems there are a lot of bot commits in history, can you squash those please. Then I think this will be ready.

**Review by sbejaoui — APPROVED** [review]
please squash translation commits as suggested by @patrickrwilson

**Comment by Freni-OSI** [comment]
Moved to #83 
CC: @AmmarOfficewalaSerpentCS  @patrickrwilson

**Pull request closed without merging**